alerts.json

```diff
@@ -0,0 +1,113 @@
+{
+  "CDAP": {
+    "CDAP_KAFKA": [
+      {
+        "name": "cdap_kafka_process",
+        "label": "CDAP Kafka Process",
+        "description": "This alert is triggered if the CDAP Kafka processes cannot be confirmed to be up and listening on the network for the configured critical threshold, given in seconds.",
+        "interval": 1,
+        "scope": "ANY",
+        "source": {
+          "type": "PORT",
+          "uri": "{{cdap-site/kafka.bind.port}}",
```

Comment:
should this be ``kafka.bind.address``?  or does it assume localhost if only a port is given?